FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Ankit on 30/07/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+internal class ManagedFeedImage: NSManagedObject {
+	@NSManaged internal var id: UUID
+	@NSManaged internal var feed_description: String?
+	@NSManaged internal var location: String?
+	@NSManaged internal var url: URL
+	@NSManaged internal var cache: ManagedCache
+
+	var local: LocalFeedImage {
+		return LocalFeedImage(id: id, description: feed_description, location: location, url: url)
+	}
+
+	internal static func image(from localFeed: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
```

Comment:
`internal` is a default access level, no need to declare it explicitly. Please remove `internal` modifier.
Also this method returns a collection so the name can be in plural to better describe intent

```suggestion
	static func images(from localFeed: [LocalFeedImage], in context: NSManagedObjectContext) -> NSOrderedSet {
```